src/backend/distributed/operations/citus_tools.c

```diff
@@ -30,8 +33,10 @@
 
 /* simple query to run on workers to check connectivity */
 #define CONNECTIVITY_CHECK_QUERY "SELECT 1"
+#define CONNECTIVITY_CHECK_COLUMNS 5
 
 PG_FUNCTION_INFO_V1(citus_check_connection_to_node);
+PG_FUNCTION_INFO_V1(citus_check_cluster_node_health);
```

Comment:
Minor: Should we move these new UDFs into own source code file? I don't think `master_run_on_worker` is very much related to what we add here. Probably best to do with a follow-up PR.

Reply:
I should do that after merging this PR. This is a really good advice.

The connectivity checks used to depend on the static functions here. Now it does not really make sense to have them here.